src/libasr/ASR.asdl

```diff
@@ -88,7 +88,7 @@ symbol
         abi abi, access access, deftype deftype, string? bindc_name, bool pure,
         bool module)
     | Function(symbol_table symtab, identifier name, expr* args, stmt* body,
-        expr return_var, abi abi, access access, deftype deftype,
+        expr return_var, abi abi, access access, deftype deftype, bool elemental,
```

Comment:
Note: a subroutine can also be elemental, but let's do that later.

Reply:
Yes. Planning to do that later. Didn't want to make changes which were out of scope.